packages/relay/src/lib/eth.ts

```diff
@@ -1651,18 +1652,26 @@ export class EthImpl implements Eth {
 
     this.contractCallFormat(call);
 
+    const selector = getFunctionSelector(call.data!);
+
+    const shouldForceToConsensus =
+      process.env.ETH_CALL_FORCE_TO_CONSENSUS_BY_SELECTOR == 'true' &&
+      constants.ETH_CALL_SELECTORS_ALWAYS_TO_CONSENSUS.indexOf(selector) !== -1;
+
+    // ETH_CALL_DEFAULT_TO_CONSENSUS_NODE = false enables the use of Mirror node
+    const shouldDefaultToConsensus = !(
+      process.env.ETH_CALL_DEFAULT_TO_CONSENSUS_NODE === undefined ||
+      process.env.ETH_CALL_DEFAULT_TO_CONSENSUS_NODE == 'false'
+    );
```

Comment:
I believe this would equivalent to below
```suggestion
     const shouldDefaultToConsensus = process.env.ETH_CALL_DEFAULT_TO_CONSENSUS_NODE === 'true'
```